src/cljs/rems/fields.cljs

```diff
@@ -15,6 +15,19 @@
 (defn field-name [field]
   (str "form-" (getx field :form/id) "-field-" (getx field :field/id)))
 
+(defn collapse [info-id info-text focus-when-collapse-opened body-text]
```

Comment:
Should add docstring and use keyword arguments here.

Reply:
> docstring

this is the only one unresolved I managed to find

Reply:
Keyword arguments too.